.github/advanced-issue-labeler.yml

```diff
@@ -6,14 +6,14 @@ policy:
     - id: ['versions']
       label:
         - name: "ver: 1.6.x"
-          keys: ['v1_6']
+          keys: ['v1_6', 'v1.6', '1.6.x']
         - name: "ver: 1.7.x"
-          keys: ['v1_7']
+          keys: ['v1_7', 'v1.7', '1.7.x']
         - name: "ver: 1.8.x"
-          keys: ['v1_8']
+          keys: ['v1_8', 'v1.8', '1.8.x']
         - name: "ver: 1.9.x"
-          keys: ['v1_9']
+          keys: ['v1_9', 'v1.9', '1.9.x']
         - name: "ver: 2.0.x"
-          keys: ['v2_0']
+          keys: ['v2_0', 'v2.0', '2.0+']
```

Comment:
Looks correct.

> **Note**: The array of `keys` should correspond to your options in the dropdown inside your issue form template. Since your template doesn't have options like `2.0+`, you don't have to specify them in the configuration unless you want to add them to the dropdown later.